.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -101,23 +108,28 @@ https://gitlab.grml.org/grml/build-release/-/pipelines/TBD/builds
     sha256sum -c SHA256SUMS-20YY.MM
     gpg --keyid-format long --verify SHA256SUMS-20YY.MM.gpg SHA256SUMS-20YY.MM
     ```
-  - [ ] upload `grml*iso grml*tar` (best done via curl directly from gitlab)
-  - [ ] upload `SHA256SUMS-20YY.MM.gpg *asc`
+  - [ ] upload `grml*iso grml*tar` (best done via curl directly from gitlab) (*Comment: to where?*)
+  - [ ] upload `SHA256SUMS-20YY.MM.gpg *asc` (*Comment: to where?*)
```

Comment:
Hmm, the comment should actually be "from where?" and affects the whole "sign + upload to" section.

I suppose the files are on the Gitlab server. If yes, I would just add "from `gitlab.grml.org` in the "sign + upload to" line...